website/docs/vscode.md

```diff
@@ -0,0 +1,414 @@
+---
+id: vscode
+title: Extension for Visual Studio Code
+sidebar_label: Visual Studio Code
+---
+
+The Sorbet extension for Visual Studio Code integrates with the Sorbet language
+server to provide IDE-like features for typed Ruby files.
```

Comment:
The http://go/sorbeta docs have a bunch of other instructions about how to actually install the extension. Do you think that's worth mentioning in this doc?

Reply:
welp, i see that's actually the second section. what do you think about swapping the features and installing section?

Reply:
done